roles/openshift_logging/README.md

```diff
@@ -12,7 +10,156 @@ generation for Elasticsearch (it uses JKS) as well as openssl to sign certificat
 As part of the installation, it is recommended that you add the Fluentd node selector label
 to the list of persisted [node labels](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-node-host-labels).
 
-### Required vars:
+### Change to variable structure and deployment structure:
+
+Previously, the EFK stack was deployed as either an all-in-one or a split (apps and ops) deployment based on the boolean variable
+`openshift_logging_use_ops`. This has been phased out in favor of being able to configure multiple (currently up to two is supported)
+ES clusters.
+
+To be able to accomplish this, the old variable naming structure has been revamped to follow a dictionary structure as such:
+
+```yaml
+openshift_logging:
+  cluster_names: ['es']
+  fluentd:
+    nodeselector: {'logging-infra-fluentd': 'true'}
+    cpu_limit: null
+    cpu_request: 100m
+    memory_limit: 512Mi
+    journal_source: ""
+    journal_read_from_head: ""
+    hosts: ['--all']
+    buffer_queue_limit: 32
+    buffer_size_limit: 8m
+    app:
```

Comment:
Represents config for the app pipeline? and the Infra pipeline?

Reply:
This is so we can specify the end points and certs for fluentd, yes.

Reply:
imho `app` is not a good name here. I spent some time trying to understand what it is: "what has app to do with fluentd?". Maybe a longer more descriptive name?

I don't know if it adds a lot of value to specify output in such form. My understanding is that key scenarios are: output to ES(app, infra), output to mux(secure_forward). How would I specify that I want to potentially send `app` logs to mux? Also app and infra certs are pretty much hardcoded, maybe just have a boolean or something instead of full paths to certificates, as the certs won't be changing much? i.e. I think this is too configurable for my taste.

Reply:
So my thought here is that we want to give users a way to specify the endpoint, which they could easily point to mux this way. So any application logs would go to mux, or any infra logs could, or both.

I agree with the cert paths comment, those shouldn't change actually... they will always be the same within the pod from the secret.